examples/preprocessing/complex_shape_2d.jl

```diff
@@ -0,0 +1,15 @@
+using TrixiParticles
+
+particle_spacing = 0.05
+
+file = "hexagon"
+filename = joinpath("examples", "preprocessing", file * ".asc")
```

Comment:
```suggestion
filename = "hexagon"
file = joinpath("examples", "preprocessing", file * ".asc")
```
I would switch those names.